lib/libc/Kconfig

```
@@ -138,6 +138,13 @@ config MINIMAL_LIBC_LL_PRINTF
 	  Build with long long printf enabled. This will increase the size of
 	  the image.
 
+config MINIMAL_LIBC_OPTIMIZE_STRING_FOR_SIZE
+	bool "Use size optimized string functions"
+	default n
```

I think it would be sensible to enable this option by default if the global "optimise for size" option is selected:
```suggestion
	default y if SIZE_OPTIMIZATIONS
```

---

Done.